dist/redhat_access_angular_ui_common.js

```diff
@@ -713,6 +713,16 @@ angular.module('RedhatAccess.header').directive('rhaHeader', function () {
 });
 
 'use strict';
+/*jshint unused:vars */
```

Comment:
@renujhamtani not sure why we are adding under the dist folder, this should generally get created unless we don't have the build structure. Although i remember Spenser mentioning something like compile and push.